templates/index.jsx

```diff
@@ -26,14 +26,13 @@ export default class Index extends React.Component {
           <meta name="apple-mobile-web-app-capable" content="yes" />
 
           {/* Polyfills before CSS */}
-          <meta name="ie-shiv-polyfill" dangerouslySetInnerHTML={{__html:
+          <span dangerouslySetInnerHTML={{__html:
```

Comment:
The new version of react does not like using `dangerouslySetInnerHTML` inside a `meta` tag? Is there a better alternative to this? Maybe doing `<script src="${CLOUDFLARE}/html5shiv/3.7.2/html5shiv-printshiv.js"></script>` without a `meta` tag?